README.md

```
@@ -444,17 +492,30 @@ await openID4VP.sendErrorToVerifier(error: AuthorizationConsent.consentRejectedE
+
 ## 🚨 Deprecation Notice
 
 The following methods are deprecated and will be removed in future releases. Please migrate to the suggested alternatives.
 
-| Method Name         | Description                               | Deprecated Since | Suggested Alternative                                       |
-|---------------------|-------------------------------------------|------------------|-------------------------------------------------------------|
-| sendErrorToVerifier | Sends Authorization error to the verifier | 0.6.0            | [sendErrorResponseToVerifier](#sendErrorResponseToVerifier) |
+| Method Name                 | Description                                   | Deprecated Since | Suggested Alternative                                                       |
+|-----------------------------|-----------------------------------------------|------------------|-----------------------------------------------------------------------------|
+| shareVerifiablePresentation | Sends VP (Authorization response) to verifier | 0.6.0            | [sendAuthorizationResponseToVerifier](#sendauthorizationresponsetoverifier) |
+| sendErrorToVerifier         | Sends Authorization error to the verifier     | 0.6.0            | [sendErrorResponseToVerifier](#sendErrorResponseToVerifier)                 |
```

markdownlint-cli2 (0.18.1)

Link fragments should be valid
Expected: #senderrorresponsetoverifier; Actual: #sendErrorResponseToVerifier

(MD051, link-fragments)